### DISCUSSION TITLE
Configurable splitter width

### DISCUSSION THREAD

**Discussion by MaximBG** [johnwalley/allotment#798] [Q&A]
Is it possible to change the splitter width?
If not, please consider this as a feature request ;-)

**Comment by DipeshRajoria007** [comment]
I've implemented something similar by targeting the splitter's CSS class directly.
<img width="579" alt="Screenshot 2025-06-15 at 11 18 46 PM" src="https://github.com/user-attachments/assets/fcd8c030-3586-4dcf-82af-e64a75f1912f" />

You can try applying custom styles by targeting these specific classes:
<img width="697" alt="Screenshot 2025-06-15 at 11 19 28 PM" src="https://github.com/user-attachments/assets/db3f0d62-9bb7-44ea-9925-1eba07fda87f" />

Hope this helps!
